docs/getting_started/requirements.md

```diff
@@ -33,7 +33,7 @@ These setups are tested by QA and are generally recommended setups. For security
 |------|------|
 |Search|Solr (recommended, for better performance and scalability of all API Queries):</br></br>Solr 4.10</br>*Solr 6 SOLR BUNDLE >= 1.3, CURRENTLY TESTED WITH SOLR 6.6LTS*</br></br>Oracle Java/Open JDK: 7 or 8 (needed for Solr, version 8 recommended)</br></br>*NOTE: If your Java version is higher than 8, you need to downgrade it to Oracle Java/Open JDK: 8.* |
 |Graphic Handler|GraphicsMagick or ImageMagick or GD|
-|[Clustering](../guide/clustering.md)|Linux NFS *or* S3 *(for IO, aka binary files stored in content repository)*</br>Memcached *or* Redis 3.0 or higher *(preferably a separate volatile-ttl instance for sessions, and an allkeys-lru/allkeys-lfu instance for cache)*</br>[Varnish](http://varnish-cache.org/) 4.1 or higher with [varnish-modules](https://github.com/varnish/varnish-modules/blob/master/README.rst) (if you use varnish-modules 0.9.x, you need [ezplatform-http-cache](https://github.com/ezsystems/ezplatform-http-cache) 0.8.x) *or* [Fastly](https://www.fastly.com/) using [our bundle provided with eZ Platform Enterprise](../guide/http_cache.md#serving-varnish-through-fastly) *(for HttpCache)*|
+|[Clustering](../guide/clustering.md)|Linux NFS *or* S3 *(for IO, aka binary files stored in content repository, not supported with legacy)*</br>Memcached *or* Redis 3.0 or higher *(preferably a separate volatile-ttl instance for sessions, and an allkeys-lru/allkeys-lfu instance for cache)*</br>[Varnish](http://varnish-cache.org/) 4.1 or higher with [varnish-modules](https://github.com/varnish/varnish-modules/blob/master/README.rst) (if you use varnish-modules 0.9.x, you need [ezplatform-http-cache](https://github.com/ezsystems/ezplatform-http-cache) 0.8.x) *or* [Fastly](https://www.fastly.com/) using [our bundle provided with eZ Platform Enterprise](../guide/http_cache.md#serving-varnish-through-fastly) *(for HttpCache)*|
```

Comment:
@DominikaK I realize this is not clear if the `, not supported with legacy` applies to just S3 or NFS.
Maybe you should do small commit to change `. S3 not supported with legacy`

Reply:
If it applies to both NFS and S3, it is clear in the context to my mind (aka, I read it like this when reading your PR)

Reply:
It does *not* apply to both, only S3

Reply:
Clarified in 138e195c72f52c6db874ca6a00c48aebc3182c48